packages/nuxt/src/module.ts

```diff
@@ -31,6 +31,7 @@ const logtoModule: NuxtModule<LogtoRuntimeConfigInput> = defineNuxtModule<LogtoR
           signOut: '/sign-out',
           callback: '/callback',
         },
+        baseUrlOverride: undefined,
```

Comment:
Perhaps we could simplify the naming by changing to just `baseUrl`, similar to the Next.js implementation.

Reply:
Sorry it took me so long to get to this. I've updated the naming.